views/clients/index.erb

```diff
@@ -1,7 +1,48 @@
-<table>
-<% for @client in @clients %>
-<tr>
-	<td><%= @client.name %><td>
-</tr>
-<% end %>
-</table>
\ No newline at end of file
+
+      <div class="container">
+        <br>
+        <div class="title">
+          <h2>Clients List</h2>
+          <a href="/clients/new">Create</a>
+        </div>
+        <br>
+        <table class="table">
+          <thead>
+            <tr>
+              <th>#</th>
+              <th>Client Name</th>
+              <th>Contact Name</th>
+							<th>Contact Email</th>
+              <th>Description</th>
+              <th>Edit</th>
+            </tr>
+          </thead>
+          <tbody>
+						<% for @client in @clients %>
```

Comment:
It's looks like a tabs vs spaces issue on my Atom. I'll push a fix in my next commit.